content/en/blog/2024/spring-starter-stable/index.md

```diff
@@ -0,0 +1,693 @@
+---
+title: The OpenTelemetry Spring Boot starter is now stable
+linkTitle: Spring Starter stable
+date: 2024-09-30
+author: >
+  [Gregor Zeitlinger](https://github.com/zeitlinger) (Grafana Labs), [Jean
+  Bisutti](https://github.com/jeanbisutti) (Microsoft)
+issue: https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/11581
+sig: Java
+cSpell:ignore: Bisutti Gregor Zeitlinger
+---
+
+<!-- markdownlint-disable blanks-around-fences -->
+<?code-excerpt path-base="examples/java/spring-starter"?>
+
+We're proud to announce that the OpenTelemetry Spring Boot starter is now stable.
+
+The [Spring Boot](https://spring.io/projects/spring-boot) starter is a powerful
+tool that simplifies the process of instrumenting Spring Boot applications with
+OpenTelemetry. It provides a lightweight, flexible alternative to the
+OpenTelemetry Java agent, making it easier than ever to observe your Spring Boot
+applications.
+
+In this blog post, we will explain when you should use the Spring Starter, what it
+actually means to be stable, what are the main features, and which challenges we faced. 
+In the last part, we will demonstrate some of the starter's features using a GraalVM native image 
+application.
+
+If you just want to get started, check out the
+[Spring Starter documentation](/docs/zero-code/java/spring-boot-starter).
+
+## When to use the Spring Starter?
+
+Here are some scenarios where you might want to use the Spring Starter:
+
+- **Spring Boot Native image** applications for which the OpenTelemetry Java
+  agent does not work
+- **Startup overhead** of the OpenTelemetry Java agent exceeding your
+  requirements
+- A Java agent is already used because the OpenTelemetry Java agent
+  might not work with the other agent
+- **Spring Boot configuration files** (`application.properties`,
+  `application.yml`) to configure the OpenTelemetry Spring Boot starter which
+  doesn't work with the OpenTelemetry Java agent
+- **Programmatic configuration** of the OpenTelemetry Spring Boot starter, such
+  as
+  [dynamic auth headers](/docs/zero-code/java/spring-boot-starter/sdk-configuration/#configure-the-exporter-programmatically),
+  using Spring beans (the OpenTelemetry Java agent requires an
+  [extension](/docs/zero-code/java/agent/extensions/) for this)
+- **Uses code dependencies**: You don't need to add any JVM options (e.g. in your Docker file) - 
+  just add a dependency and a BOM to your `pom.xml` or `build.gradle` file
+
+It may be a bit surprising, but if you don't build a Spring native image application
+our default recommendation is to use the [**OpenTelemetry Java agent**](/docs/zero-code/java/agent)
+with bytecode instrumentation, as it provides more out of the box instrumentation than the 
```

Comment:
```suggestion
with bytecode instrumentation, as it provides a simpler onboarding experience and provides much more out of the box instrumentation than the 
```

Reply:
I don't agree - starters is the easier onboarding experience for spring users

Reply:
that's fair 👍